search/src/test/scala/weco/api/search/works/WorksIncludesTest.scala

```diff
@@ -571,41 +563,181 @@ class WorksIncludesTest
     ) {
       withWorksApi {
         case (worksIndex, routes) =>
-          val genres1 = List(Genre("ornithology", List(Concept("ornithology"))))
-          val genres2 = List(Genre("flying cars", List(Concept("flying cars"))))
-          val work1 = indexedWork(canonicalId = canonicalId1).genres(genres1)
-          val work2 = indexedWork(canonicalId = canonicalId2).genres(genres2)
-
-          insertIntoElasticsearch(worksIndex, work1, work2)
+          indexTestDocuments(worksIndex, worksEverything: _*)
 
-          assertJsonResponse(routes, s"$rootPath/works?include=genres") {
-            Status.OK -> s"""
-              {
-                ${resultList(totalResults = 2)},
-                "results": [
-                 {
-                   "type": "Work",
-                   "id": "${work1.state.canonicalId}",
-                   "title": "${work1.data.title.get}",
-                   "alternativeTitles": [],
-                   "availabilities": [${availabilities(
-              work1.state.availabilities
-            )}],
-                   "genres": [ ${genres(genres1)}]
-                 },
-                 {
-                   "type": "Work",
-                   "id": "${work2.state.canonicalId}",
-                   "title": "${work2.data.title.get}",
-                   "alternativeTitles": [],
-                   "availabilities": [${availabilities(
-              work2.state.availabilities
-            )}],
-                   "genres": [ ${genres(genres2)}]
-                 }
-                ]
-              }
-            """
+          assertJsonResponse(routes, path = s"$rootPath/works?include=genres") {
+            Status.OK ->
+              s"""
+                 |{
```

Comment:
IIUIC, both JSONs are parsed by assertJsonResponse and compared as objects, so preserving the "correct" indentation with `|` isn't needed.

Having the `|` prefix and stripMargin at the end makes it look like assertJsonResponse is directly comparing JSON strings, giving the impression that these tests could be a bit fragile (e.g. that it would break if we change the indentation in the API responses).

Reply:
👍

I think that's auto-added by IntelliJ so I didn't really think about it; removing it is sensible so I'll do that.

Reply:
Oh, and yes, you've understood correctly.

Reply:
I think this is sorted now, good to merge?